course/2 Neural Network Architectures/2.1 Convolutional Neural Networks/2.1.2 Common Pre-Trained Models/convnext.mdx

```diff
@@ -0,0 +1,46 @@
+
+# ConvNext - A ConvNet for the 2020s (2022)
+
+
+## Introduction
+
+The ConvNext architecture was developed in 2022 by researchers from Facebook AI Research (Zhuang Liu, Hanzi Mao, Chao-Yuan Wu, Christoph Feichtenhofer, Trevor Darrell, and Saining Xie). 
+Recently, with the breakthrough of Visual Transformers (ViTs), it quickly superceded pure ConvNet models as the new state-of-the-art for Image Recognition.
```

Comment:
A line can be added to explain that "ConvNeXt is nothing more than the modernization of a standard ResNet toward the design of a vision transformer, while also improving several performance differences along the way. The result of this investigation is the ConvNeXt family of pure ConvNet models."